params/protocol_params.go

```diff
@@ -170,7 +170,7 @@ const (
 	Bls12381MapG2Gas          uint64 = 75000 // Gas price for BLS12-381 mapping field element to G2 operation
 
 	P256VerifyGas uint64 = 3450 // secp256r1 elliptic curve signature verifier gas price
-	GpgEd25519VerifyGas uint64 = 2000 // GPG Ed25519 signature verification gas price
+	GpgEd25519VerifyGas uint64 = 7900 // GPG Ed25519 signature verification gas price
```

Comment:
Assuming Ecrecover is fairly priced (as per [EIP-152 Blake2 precompile](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-152.md)) at `3000 gas/op` for ~`50,100 ns/op`.

GpgEd25519Verify should be priced proportionately `7900 gas/op` for ~`134,000 ns/op` as a good first estimate for gas cost.